credentials/apps/credentials/tests/factories.py

```diff
@@ -10,6 +12,13 @@
 PASSWORD = "dummy-password"
 
 
+def create_test_image(extension):
```

Comment:
you are already into the package `tests` so just `create_image`, use namespaces wisely. And typing!

Reply:
[Fixed](https://github.com/raccoongang/credentials/pull/1/commits/f6c2702b13dee648d3963989eb30553948555d4d).